libs/MeshKernel/src/Mesh2D.cpp

```diff
@@ -2019,3 +2019,39 @@ meshkernel::Mesh2D Mesh2D::Merge(const Mesh2D& mesh1, const Mesh2D& mesh2)
 
     return mergedMesh;
 }
+
+meshkernel::BoundingBox Mesh2D::GetBoundingBox() const
+{
+
+    Point lowerLeft(std::numeric_limits<double>::max(), std::numeric_limits<double>::max());
+    Point upperRight(-std::numeric_limits<double>::max(), -std::numeric_limits<double>::max());
+
+    const auto numNodes = GetNumNodes();
```

Comment:
Use in L. 2030 directly, no need to declare and assign numNodes. Even better, use range-based for loop:
```
for (auto const& node : m_nodes)
{
    lowerLeft.x = std::min(lowerLeft.x, nodes.x);
    lowerLeft.y = std::min(lowerLeft.y, nodes.y);
    upperRight.x = std::max(upperRight.x, nodes.x);
    upperRight.y = std::max(upperRight.y, nodes.y);
}
```